docs/src/quickstart/index.md

```diff
@@ -1,35 +1,48 @@
 # Quickstart
 
-A cursory explanation on how to get up and running with an index in 5 minutes
+In this Quickstart tutorial we'll go over:
 
-> This Quickstart will assume that you've:
->
-> - Read over [Getting Started](./../getting-started/index.md)
-> - Have installed all relevant [system](./../getting-started/system-dependencies.md) dependencies
-> - Have installed all relevant [application](./../getting-started/application-dependencies.md) dependencies
-> - Have already created a Fuel project according to [the recommended project structure](./../getting-started/fuel-indexer-project.md)
-> - Have installed the [`forc index`](http://localhost:3000/plugins/forc-index.html) plugin via [`fuelup`](http://localhost:3000/getting-started/application-dependencies/fuelup.html)
+1. How to bootstrap an environment for use with a Fuel indexer.
+2. How to use the `forc index` plugin to create, build, and deploy indicies.
+3. How to query our newly created indices for data using GraphQL.
+
+> IMPORTANT: Docker is a prerequisite for using this Quickstart. If Docker is not installed on your machine, please review the Docker installation instructions [here](https://docs.docker.com/engine/install/).
+
+## 1. Setting up your environment
 
-## What we'll do
+As a user of the Fuel indexer, you shouldn't require _too many_ dependencies. The indexer binaries available via `fuelup` are precompiled with all required dependencies, and optimized for size.
 
-In this Quickstart we're going to write a simple Sway 🌴 smart contract, then
-build and deploy and index that saves events from this contract into a database.
+In this Quickstart, we'll only be using Postgres. Further, instead of going through the
+drudgery of installing Postgres on your local machine, we'll just use [the `latest`
+Postgres docker image](https://hub.docker.com/_/postgres) via the following command: `docker pull postgres`.
 
-- [Writing a contract](#writing-a-contract)
-- [Create and deploy an index](#create-and-deploy-an-index)
-  - [Initializing an index](#1-initialize-a-new-index-project)
-  - [GraphQL data models](#2-graphql-data-models)
-  - [Update index manifest](#3-update-index-manifest)
-  - [Write your index code](#4-write-your-index-code)
-  - [Compile the index](#5-compile-the-index)
-  - [Start the index service & deploy your index](#6-start-the-service--deploy-your-index)
+## 2. Using the `forc-index` plugin
```

Comment:
@hrryndrsn 
- Same question as was posed [here](https://github.com/FuelLabs/fuel-indexer/pull/486#discussion_r1081467061) regarding `forc-indexer`
- I didn't mention `fuel-indexer` (other than a brief blurb) because we aren't actually using the binary in this Quickstart -- we're using the docker image.

Reply:
this comment is related the usage of the word "plugin" in user facing documention.

Reply:
`forc-index` is actually a plugin for `forc`, as stated in the sentences immediately preceding this one, so I don't necessarily think it's a faux pas to refer to it as such. Personally, I'm fine with either how it is now or changing to "Using `forc index`" as that's the general command invocation that we use in the book.

In any case, as we directly refer to `forc` in the sentences following this one, I think it may be prudent to add additional information to the `fuelup` step that states `forc` will be installed so the user knows what `forc` is before we mention that `forc-index` is a plugin for it.